.gitignore

```diff
@@ -1,6 +1,7 @@
 /node_modules
 /public/hot
 /public/storage
+/public/modules/*
```

Comment:
Is this needed?

Reply:
Yes, again when we publish module, it just copied file from the module assets to public, we dont need I guess. Take a final call and I will remove it if there is intention to copy assets from modules to public.